pandas/tests/groupby/test_aggregate.py

```diff
@@ -822,7 +822,14 @@ def test_cython_agg_empty_buckets(self):
         grps = range(0, 55, 5)
 
         for op, targop in ops:
-            result = df.groupby(pd.cut(df[0], grps))._cython_agg_general(op)
+            # calling _cython_agg_general directly, instead of via the user API
```

Comment:
This bit was unfortunate, but I think necessary since we're calling _cython_agg_general directly.

I'll need change `min_count` to 0 in the followup PR.